### DISCUSSION TITLE
Use markdownlint rules

### DISCUSSION THREAD

**Discussion by RemcoSchrijver** [streamdeck-linux-gui/streamdeck-linux-gui#30] [Ideas / Feature Requests]
Right now the documentation is not as nicely formatted as you would expect of raw markdown. It might be a good idea to enforce markdownlint rules? I would be willing to get all the current markdown files in a passing state. As well as adding a workflow for verifying that new documentation follows the same formatting.

**Comment by Data-007** [comment]
The Arch install ones I had a hand in doing, in parts I messed up and did not have the opportunity to make good on the mistakes I had maid they(Install docs) need to be accurate. 
The actual readme file I cant see why it should not be upgraded and screenshots  redone. You could just fork the repo like I did make the changes and submit a pull request on develop branch. We also had someone else interested in documentation to.

**Comment by coolapso** [comment]
> Right now the documentation is not as nicely formatted as you would expect of raw markdown. It might be a good idea to enforce markdownlint rules? I would be willing to get all the current markdown files in a passing state. As well as adding a workflow for verifying that new documentation follows the same formatting.

Sounds good, I'd be happy go review and approve it,  or I might even do it myself at some point if you don't come forward with it.

**Comment by RemcoSchrijver** [comment]
#44 Closed with this PR
